container/config.py

```diff
@@ -76,7 +76,6 @@ def project_name(self):
         return os.path.basename(self.base_path)
 
     @property
-    @abstractproperty
     def image_namespace(self):
```

Comment:
Shouldn't need to remove @abstractproperty. The intention is that every engine provide an implementation of the `image_namespace` property.